### PR TITLE
Add counting letters

### LINKED ISSUE #3: [Student A] Count letters
Check instructions from the doc

### PR DESCRIPTION
**Pull request by Ostrokrzew**
Add method for counting letters in given file.
Counts only letters from 'A' to 'Z', upper or lower case doesn't matter.
Doesn't count non-latin letters.
This also closes #3 issue.

Signed-off-by: Ostrokrzew <ostrokrzew@protonmail.com>

### CODE REVIEW

**Comment by Ostrokrzew** [comment]
Please review before merge.

**Review by 1929940 — COMMENTED** [review]
looks good

**Review by 1929940 — COMMENTED** [review]
Id go with Char.IsLetter() but Regex works too.

https://docs.microsoft.com/en-us/dotnet/api/system.char.isletter?view=netframework-4.8

File: AppMenu.cs [hunk 1]
```diff
@@ -119,6 +124,16 @@ public static void MainMenu()
             } while (cndt != 8);
         }
 
+        static int CountLetters(StreamReader file)
+        {
+            string fileReaded = file.ReadToEnd();
```
Comment (1929940) [inline AppMenu.cs]: fileRead*

File: AppMenu.cs [hunk 2]
```diff
@@ -119,6 +124,16 @@ public static void MainMenu()
...
+            Regex regex = new Regex(@"[^A-Za-z]");
```
Comment (1929940) [inline AppMenu.cs:130]: Char.IsLetter() would be more clear but regex is fine too

File: AppMenu.cs (hunk 1 above)
Comment (1929940) [inline AppMenu.cs:129]: fileRead

**Review by 1929940 — COMMENTED** [review]
No issues worth delaying the merge.

**Review by 1929940 — COMMENTED** [review]
Nothing to add. Looks good.

**Pull request merged**